includes/classes/Indexable/Post/Post.php

```diff
@@ -771,7 +771,15 @@ public function format_args( $args, $wp_query ) {
 		 *
 		 * @since  3.4
 		 */
-		$taxonomies = get_taxonomies();
+
+		/**
+		 * Filter taxonomies used for generating tax query
+		 *
+		 * @hook ep_post_get_taxonomies
+		 * @param  {array} $taxonomies an array of taxonomies
+		 * @return  {array} List of taxonomies to use for tax query
+		 */
+		$taxonomies = apply_filters( 'ep_post_get_taxonomies', get_taxonomies() );
```

Comment:
The Post indexable has a method that we can probably reuse here - `get_indexable_post_taxonomies()` - it's used when generating an ES doc for indexing...so presumably it would make sense to limit which taxonomies we try to parse out of the args to the same ones?

Does introduce a bit of a disconnect between the `WP_Query()` and the _actual_ ES query that gets created, but I think if it's limited to being used in only this part that tries to parse out taxonomies from the root of the WP_Query, it should be safe.

Aside - `get_indexable_post_taxonomies()` also has a similar filter - `ep_sync_taxonomies`.

Reply:
@nickdaugherty 

Sure, makes sense. I was doing the change in a way to get it merged upstream with minimal changes(they just use get_taxonomies). 

I'll update here and then apply it to mu-plugins after :+1:

Reply:
Actually not so sure that's the best way, because ideally we can still allow those taxonomies to be indexed (not filter them out). That would be particularly important in the case of Edit Flow (where the `post_status` taxonomy comes from).

Instead, we can modify this behavior to not interpret any "core" query arguments as taxonomies...so basically have a list of the known WP_Query vars (like `post_status`) and then EP doesn't try to interpret them as taxonomies, even if there is a taxonomy with that name.

Reply:
Full list of core params: https://developer.wordpress.org/reference/classes/wp_query/#parameters